blockchain/chain_sync/Cargo.toml

```diff
@@ -28,35 +28,35 @@ forest_networks        = "0.1.0"
 forest_state_manager   = "0.1.0"
 forest_statediff       = { version = "0.1.0", optional = true }
 forest_vm              = "0.3.1"
-futures                = "0.3.5"
-futures-util           = "0.3.5"
-fvm                    = "1.0"
+futures                = "0.3.24"
+futures-util           = "0.3.24"
+fvm                    = "1.1"
 fvm_ipld_encoding      = "0.2"
 fvm_shared             = { version = "0.8.0", default-features = false }
 lazy_static            = "1.4.0"
-log                    = "0.4.8"
-lru                    = "0.7.2"
+log                    = "0.4.17"
+lru                    = "0.7.8"
 nonempty               = "0.8.0"
 num-bigint             = "0.4"
 num-traits             = "0.2"
-pbr                    = "1.0.3"
-prometheus             = { version = "0.12.0", features = ["process"] }
+pbr                    = "1.0.4"
+prometheus             = { version = "0.13.2", features = ["process"] }
 quickcheck             = { version = "1" }
-rand                   = "0.7.3"
+rand                   = "0.8.5"
 serde                  = { version = "1.0", features = ["derive", "rc"] }
-smallvec               = "1.1.0"
+smallvec               = "1.9.0"
 thiserror              = "1.0"
 time                   = { version = "0.3", features = ["serde"] }
-tokio                  = { version = "1.0", features = ["sync"] }
+tokio                  = { version = "1.21", features = ["sync"] }
 
 [dev-dependencies]
```

Comment:
can we clean this up a bit? I mean, at least remove duplicates, i.e. packages that are already included in `[dependencies]` like `pbr`.

Reply:
Done